src/components/keyboard-aware-container.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+import React, { Component } from 'react';
+import { View, Keyboard, Platform } from 'react-native';
+
+type Props = {
+  onKeyboardStateChange: (state: string, event: any) => void,
+  style?: Object | Array<Object>,
+  children: Object,
+};
+
+export class KeyboardAwareContainer extends Component<Props> {
+  constructor(props) {
+    super(props);
+    this.keyboardWillShowSub = null;
+    this.keyboardWillHideSub = null;
```

Comment:
Do the right flow type annotations like following? Sorry that I'm a flow expert and I don't know how to import `EmitterSubscription`.

```
  keyboardWillShowSub: EmitterSubscription;
  keyboardWillHideSub: EmitterSubscription;
```

By the way, to make eslint happy, we need update our `.eslintrc`.

```
    "react/sort-comp": [
      "error",
      {
        "order": [
          "static-methods",
 +        "type-annotations",
```